### ISSUE TITLE
About training using resnet18 with imagenet

### ISSUE THREAD

**Issue by CheungBH** [huawei-noah/GAN-pruning#21]
Hi. Thanks for your work. I am currently running your code training with imagenet. It runs, but when the conducting "validate_ontrain 
https://github.com/yehuitang/Pruning/blob/5c8bc0d780fab41e1bd894b0360bd50e14cd0571/ManiDP/main.py#L487
", the loss becomes enormous. 
![image](https://user-images.githubusercontent.com/39186017/147448700-b6af5e1e-e735-45e7-8100-d1f3b31b7c52.png)

And the loss becomes normal after training the first epoch, but the remain rate you displayed is large even training for hundreds of epochs.
<img width="1354" alt="Screenshot 2021-12-27 at 15 50 20" src="https://user-images.githubusercontent.com/39186017/147448548-7466894c-430b-4291-9ce8-ce829c3c5ae2.png">
 
Is it normal? I have loaded the pretrained model

**Comment by CheungBH** [comment]
The training seems to be normal when setting the clamp_max to a lower value.
10 for dyresnet20, and 1 for dyresnet56
